interfaces/cython/SConscript

```diff
@@ -134,9 +139,9 @@ if localenv['PYTHON_INSTALLER'] == 'direct':
         for filename in open('build/python-installed-files.txt').readlines():
             filename = filename.strip()
             if filename.endswith(check):
-                filename = filename.replace(check, '')
-                global_env['python_module_loc'] = normpath(filename)
+                global_env["python_module_loc"] = normpath(filename.replace(check, ""))
                 break
+
```

Comment:
It looks like you've reverted the changes from 5720ff6, so you may want to drop that commit altogether, to avoid a confusing history.